src/components/Rating/Rating.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react';
+import { RadioButton as CoreRadio } from 'wix-ui-core/radio-button';
+import styles from './Rating.st.css';
+import { ReactComponent as StarIcon } from '../../assets/icons/Star.svg';
+
+export enum IconSize {
+  Small = 'small',
+  Large = 'large',
+}
+
+export enum Mode {
+  Input = 'input',
+  Display = 'display',
+}
+
+export interface RatingProps {
+  value: number;
+  mode: Mode;
+  onSelect?(value: number): void;
+  disabled?: boolean;
+  error?: boolean;
+  iconSize?: IconSize;
+}
+
+interface DefaultProps {
+  value: number;
+  disabled: boolean;
+  error: boolean;
+  mode: Mode;
+  iconSize: IconSize;
+}
+
+/** Rating component based on IconToggle */
+export class Rating extends React.Component<RatingProps> {
```

Comment:
I know it's super super picky, but I talked to the designers, and they say, that in order to be consistent with the editor, this component should be called `Ratings` and not `Rating` 🤦‍♂ .
Sorry for not checking earlier.

Reply:
No worries